src/web100srv.c

```diff
@@ -2614,12 +2446,57 @@ int main(int argc, char** argv) {
                   case 'S':
                     SysLogFacility = optarg;
                     break;
+                  case 314:
```

Comment:
Not really.  Magic numbers is how getopt_long() keeps track of which argument is which.

When you pass the argument structure into getopt_long, you need to assign a magic number to every argument. Then, you have to switch on the returned number and have a case for every magic number you passed in.  In the past, some arguments used single chars as the magic numbers (viz. the 'S' in the previous case), but now web100srv supports so many args that the char namespace is overcrowded.